app/src/main/res/layout-land/profile_chooser_add_view.xml

```diff
@@ -40,6 +40,7 @@
         android:layout_width="72dp"
         android:layout_height="72dp"
         android:layout_marginBottom="8dp"
+        android:importantForAccessibility="no"
```

Comment:
Just to check: is this because the overall layout is clickable and has the correct content description? / Why is CircleImageView even picked up by the accessibility scanner? I thought ImageViews are ignored by default unless they have a content description or are clickable.

Reply:
@rt4914 while I am using talkback on this screen it is not showing picking up circular image view as a separate item, rather it is picking the whole linear layout with correct description. Although this change will not cause any difference in my case. Can you verify this problem

Reply:
The reason why I added this was to be more explicit. But now as @BenHenning has mentioned `I thought ImageViews are ignored by default unless they have a content description or are clickable.` which is confirmed on scanners and reader too. I am reverting it.